editor/filesystem_dock.cpp

```diff
@@ -1553,50 +1558,30 @@ void FileSystemDock::_update_resource_paths_after_move(const HashMap<String, Str
 
 		r->set_path(base_path + extra_path);
 	}
-
-	for (int i = 0; i < EditorNode::get_editor_data().get_edited_scene_count(); i++) {
-		String file_path;
-		if (i == EditorNode::get_editor_data().get_edited_scene()) {
-			if (!get_tree()->get_edited_scene_root()) {
-				continue;
-			}
-
-			file_path = get_tree()->get_edited_scene_root()->get_scene_file_path();
-		} else {
-			file_path = EditorNode::get_editor_data().get_scene_path(i);
-		}
-
-		if (p_renames.has(file_path)) {
-			file_path = p_renames[file_path];
-		}
-
-		if (i == EditorNode::get_editor_data().get_edited_scene()) {
-			get_tree()->get_edited_scene_root()->set_scene_file_path(file_path);
-		} else {
-			EditorNode::get_editor_data().set_scene_path(i, file_path);
-		}
-	}
 }
 
-void FileSystemDock::_update_dependencies_after_move(const HashMap<String, String> &p_renames) const {
+void FileSystemDock::_update_dependencies_after_move(const HashMap<String, String> &p_renames, const Vector<String> &p_remaps) const {
 	// The following code assumes that the following holds:
 	// 1) EditorFileSystem contains the old paths/folder structure from before the rename/move.
 	// 2) ResourceLoader can use the new paths without needing to call rescan.
-	Vector<String> remaps;
-	_find_remaps(EditorFileSystem::get_singleton()->get_filesystem(), p_renames, remaps);
-	for (int i = 0; i < remaps.size(); ++i) {
+	Vector<String> scenes_to_reload;
```

Comment:
```suggestion
	List<String> scenes_to_reload;
```
In cases where you only add at the end and don't need to access random elements, a list is better.